test/src/main/java/org/jahia/modules/external/test/configuration/ConfigurationTest.java

```diff
@@ -0,0 +1,163 @@
+
+    @Before public void setUp() {
+        mountPointConfigFactory = BundleUtils.getOsgiService(ManagedServiceFactory.class, "(service.pid=org.jahia.modules.external.mount)");
+        JCRSessionFactory.getInstance().closeAllSessions();
+        properties = new Hashtable();
+    }
+
+    @Test public void testCreate() throws ConfigurationException, RepositoryException {
+        JCRNodeWrapper mountPoint = createTestMountPoint();
+
+        deleteNode(mountPoint.getIdentifier());
+    }
+
+    @Test(expected = RepositoryException.class) public void testUpdateWithoutTypeProperty()
+            throws ConfigurationException, RepositoryException {
+        properties.put(CONFIGURATION_MOUNT_J_NODENAME, "test");
+        properties.put(CONFIGURATION_MOUNT_J_ROOT_PATH, J_ROOT_PATH_VALUE);
+        properties.put(SERVICE_PID, PID);
+
+        mountPointConfigFactory.updated(PID, properties);
+        session = JCRSessionFactory.getInstance().getCurrentUserSession();
+        session.getNode(TEST_MOUNT_PATH);
+    }
+
+    private JCRNodeWrapper createTestMountPoint() throws ConfigurationException, RepositoryException {
+        properties.put(CONFIGURATION_MOUNT_J_NODENAME, "test");
+        properties.put(CONFIGURATION_MOUNT_J_ROOT_PATH, J_ROOT_PATH_VALUE);
+        properties.put(CONFIGURATION_MOUNT_JCR_PRIMARY_TYPE, JNT_VFS_MOUNT_POINT_TYPE);
+        properties.put(SERVICE_PID, PID);
+
+        mountPointConfigFactory.updated(PID, properties);
+
+        session = JCRSessionFactory.getInstance().getCurrentUserSession();
+        JCRNodeWrapper mountPoint = session.getNode(TEST_MOUNT_PATH);
+        assertEquals(J_ROOT_PATH_VALUE, mountPoint.getPropertyAsString("j:rootPath"));
+        assertEquals(JNT_VFS_MOUNT_POINT_TYPE, mountPoint.getPropertyAsString("jcr:primaryType"));
+        assertEquals("test-mount", mountPoint.getPropertyAsString(J_NODENAME_PROPERTY));
+        assertTrue(mountPoint.isNodeType("jmix:configPid"));
+        assertEquals(PID, mountPoint.getPropertyAsString("configPid"));
+        return mountPoint;
+    }
+
+    @Test public void testUpdate() throws ConfigurationException, RepositoryException {
+        createTestMountPoint();
+
+        properties = new Hashtable<>();
+        properties.put(CONFIGURATION_MOUNT_J_NODENAME, "test-renamed");
+        properties.put(CONFIGURATION_MOUNT_J_ROOT_PATH, J_ROOT_PATH_VALUE);
+        properties.put(CONFIGURATION_MOUNT_JCR_PRIMARY_TYPE, JNT_VFS_MOUNT_POINT_TYPE);
+        properties.put(SERVICE_PID, PID);
+
+        mountPointConfigFactory.updated(PID, properties);
+        JCRNodeWrapper mountPoint = session.getNode("/mounts/test-renamed-mount");
+        assertEquals("test-renamed-mount", mountPoint.getPropertyAsString(J_NODENAME_PROPERTY));
+
+        deleteNode(mountPoint.getIdentifier());
+    }
+
+    @Test public void testDelete() throws ConfigurationException, RepositoryException {
+        properties.put(CONFIGURATION_MOUNT_J_NODENAME, "test");
```

Comment:
You are missing to create the mount point. Each test must be isolated (or declare the requirements) and do clean up.